daisy_integration_tests/scripts/post_translate_test.sh

```diff
@@ -135,6 +135,14 @@ function check_google_cloud_sdk {
     fi
   fi
 
+    # Skip for Debian 8: b/161567418
```

Comment:
minor: change the bug link (which is not accessible outside Google) with a description (or have both)

Reply:
done!